packages/icons/src/svg/logo.svg

```diff
@@ -1,24 +1,3 @@
-	</g>
-	<g class="talend-logo value1">
-		<path class="talend-logo line1" d="M27.763 16.624L19.559 6.461l.263-.205 8.204 10.163-.263.205z"/>
-		<path class="talend-logo circle1" d="M18.713 5.4a1.4 1.4 0 0 1 1.96-.002 1.347 1.347 0 0 1 0 1.929 1.405 1.405 0 0 1-1.962-.002 1.346 1.346 0 0 1 .002-1.926z"/>
-	</g>
-	<path class="talend-logo text-talend" d="M58.8 19.043a6.671 6.671 0 0 0-3.027.627 2.494 2.494 0 0 0-1.317 2.35 2.767 2.767 0 0 0 3.07 2.863 6.32 6.32 0 0 0 4.104-1.764v-4.193zm3.11 6.819a7.844 7.844 0 0 1-5.5 2.273c-3.546 0-6.14-2.313-6.14-5.878a5.258 5.258 0 0 1 3.15-5.016 11.688 11.688 0 0 1 4.822-.98l3.388-.118v-.782c0-2.431-1.555-3.45-3.786-3.45a10.192 10.192 0 0 0-5.022 1.373l-1.436-2.705a14.31 14.31 0 0 1 7.096-1.842c4.664 0 7.295 2.118 7.295 6.624v7.13a10.565 10.565 0 0 0 .959 5.25h-4.15zM38.756 9.208v-4.27L42.9 3.682v5.526h5.66l-1.033 3.213H42.9v10.032c0 1.682.638 2.35 1.875 2.35a5.359 5.359 0 0 0 2.883-.897 5.699 5.699 0 0 0 1.108 2.943 10.162 10.162 0 0 1-4.987 1.286c-3.15 0-5.022-1.765-5.022-4.781V9.208zm51.15 7.366c-.08-2.352-1.234-4.662-3.905-4.662-2.513 0-3.986 2.155-4.025 4.662zm-7.89 2.9a5.262 5.262 0 0 0 5.618 5.409 9.55 9.55 0 0 0 4.744-1.256l1.236 2.785a12.125 12.125 0 0 1-6.419 1.723c-6.574 0-9.404-4.585-9.404-9.72 0-5.602 3.149-9.678 8.25-9.678 5.141 0 8.092 3.763 8.092 9.758v.978zm-3.957 7.61a11.245 11.245 0 0 1-4.597.931c-2.908 0-4.345-1.488-4.345-4.191V1.49L73.263 0v22.687c0 1.373.478 1.763 1.396 1.763a10.391 10.391 0 0 0 1.4-.264 7.652 7.652 0 0 0 2.001 2.899zM96.905 9.209h3.805l-.054 2.367a11.165 11.165 0 0 1 6.69-2.837c2.989 0 5.897 1.568 5.897 6.663v12.34h-4.142V16.693c0-2.43-.56-4.389-3.07-4.389a8.219 8.219 0 0 0-4.982 2.234V27.74h-4.144zM133.04 22.49V.04l-.003.012V0l-4.145 1.49v7.523a13.115 13.115 0 0 0-2.75-.275c-5.725 0-10.012 3.909-10.151 9.631l-.003.036c0 .029-.003.065-.003.108-.002.046-.008.09-.008.138 0 .027.004.055.004.083-.01 1.357.234 5.702 3.563 8.065l.06.042c.13.091.265.178.405.262.034.02.065.04.096.06a7.536 7.536 0 0 0 .837.424c.021.01.043.018.066.027.162.069.327.131.496.185a.132.132 0 0 1 .023.007c.151.049.306.091.464.13.063.018.127.028.189.042.108.023.215.045.324.063.111.016.22.031.33.042a3.367 3.367 0 0 0 .425.034c.138.008.273.018.415.018a7.84 7.84 0 0 0 5.5-2.273l.677 1.88H134a10.548 10.548 0 0 1-.96-5.252zm-8.966 2.33a3.76 3.76 0 0 1-1.274-.461 3.497 3.497 0 0 1-.076-.043c-.03-.018-.06-.033-.088-.053a4.85 4.85 0 0 1-.51-.354c-.103-.098-.192-.198-.283-.299a2.442 2.442 0 0 1-.18-.194 5.262 5.262 0 0 1-.63-.903c-.043-.08-.078-.16-.118-.238-.02-.042-.044-.083-.064-.126a7.98 7.98 0 0 1-.582-4.224 8.121 8.121 0 0 1 .127-.941c.005-.025.007-.052.01-.075a7.393 7.393 0 0 1 .302-1.039 5.78 5.78 0 0 1 5.673-3.723 9.336 9.336 0 0 1 2.5.31V23.12a6.126 6.126 0 0 1-4.092 1.764 2.458 2.458 0 0 1-.715-.063z"/>
-	<ellipse class="talend-logo big-circle" cx="27.896" cy="16.536" rx="5.932" ry="5.831"/>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

Comment:
You have to keep same ratio at least, looks weird.

<img width="1178" alt="capture d ecran 2018-02-20 a 12 13 31" src="https://user-images.githubusercontent.com/18534166/36421142-a8c568c4-1637-11e8-9ae7-6d67dc223408.png">